proto/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+//! ibc-proto library gives the developer access to the Cosmos SDK IBC proto-defined structs.
```

Comment:
This comment from `ibc-proto` is not suitable for `optimism_ibc_proto`.

Reply:
Fixed by https://github.com/datachainlab/optimism-elc/commit/75f7194c272b75637b5a89d5053e36a9ed0e15d1